backend/wmg/pipeline/cube_pipeline.py

```diff
@@ -47,14 +46,14 @@ def load_data_and_create_cube(
     """
     snapshot_id = int(time.time())
     snapshot_path = path if path else pathlib.Path().resolve()
-    snapshot_path = f"{snapshot_path}/{snapshot_id}"
-    corpus_path = f"{snapshot_path}/{corpus_name}"
+    corpus_path = f"{snapshot_path}/{snapshot_id}"
 
     integrated_corpus.run(path_to_h5ad_datasets, corpus_path, extract_data)
     try:
         summary_cubes.run(corpus_path, validate_cube)
     except CubeValidationException as e:
-        upload_artifacts_to_s3(snapshot_path, "latest_validation_failed_snapshot")
+        # TODO: should we split integrated/obs/var into separate directory (not under snapshot ID)?
```

Comment:
Maybe, but not worth any effort, since ultimately we hope to build WMG off Cell Census data, replaxcing these intermediate data artifacts. For now, it helps to have these linked in case we need to troubleshoot.